src/language-js/print/call-arguments.js

```diff
@@ -0,0 +1,295 @@
+"use strict";
+
+const comments = require("../../main/comments");
+const { getLast, getPenultimate } = require("../../common/util");
+const { isNextLineEmpty } = require("../../common/util-shared");
+const {
+  hasLeadingComment,
+  hasTrailingComment,
+  isFunctionCompositionArgs,
+  isJSXNode,
+  isLongCurriedCallExpression,
+  shouldPrintComma,
+} = require("../utils");
+
+const {
+  builders: {
+    concat,
+    line,
+    hardline,
+    softline,
+    group,
+    indent,
+    conditionalGroup,
+    ifBreak,
+    breakParent,
+  },
+  utils: { willBreak },
+} = require("../../document");
+
+function printCallArguments(path, options, print) {
+  const node = path.getValue();
+  const args = node.arguments;
+
+  if (args.length === 0) {
+    return concat([
+      "(",
+      comments.printDanglingComments(path, options, /* sameIndent */ true),
+      ")",
+    ]);
+  }
+
+  // useEffect(() => { ... }, [foo, bar, baz])
+  if (
+    args.length === 2 &&
+    args[0].type === "ArrowFunctionExpression" &&
+    args[0].params.length === 0 &&
+    args[0].body.type === "BlockStatement" &&
+    args[1].type === "ArrayExpression" &&
+    !args.find((arg) => arg.comments)
+  ) {
+    return concat([
+      "(",
+      path.call(print, "arguments", 0),
+      ", ",
+      path.call(print, "arguments", 1),
+      ")",
+    ]);
+  }
+
+  // func(
+  //   ({
+  //     a,
+
```

Comment:
```suggestion
  //
```

DI think this empty line in code?